recipe/meta.yaml

```diff
@@ -38,14 +36,12 @@ requirements:
     - python
     - {{ pin_compatible('numpy') }}
     - {{ pin_compatible('libitk', min_pin='x.x', max_pin='x.x') }}
-    - root_base  # [linux]
```

Comment:
why can this be removed? We need the ROOT libraries to run? (I didn't think it was inherited from `build`)